apps/repo_poller/test/setup_worker_test.exs

```diff
@@ -105,6 +106,7 @@ defmodule RepoPoller.SetupWorkerTest do
              wait_for(fn ->
                %{workers: num} = DynamicSupervisor.count_children(PollerSupervisor)
                num == 1
+               :timer.sleep 200
```

Comment:
i think, instead of doing a timer sleep 200 here, is better to increase the waiting time `def wait_for(timeout \\ 1000, f)` so we can increase it to 2sec or more if necessary